spk/deluge/src/core.conf

```diff
@@ -49,7 +49,7 @@
   "random_port": false, 
   "listen_ports": [
     6881, 
-    6891
+    6881
```

Comment:
No need for two lines with same port number. What do you expect here?

Reply:
`listen_ports` defines a range of ports [ from_port, to_port ] so it needs two entries for a single port too.